src/com/itacademy/eshop/product/Review.java

```diff
@@ -1,17 +1,17 @@
 package com.itacademy.eshop.product;
 
 public class Review {
-    private String author;
+    private Author author;
     private int rating;
     private String comment;
 
-    public Review(String author, int rating, String comment) {
-        this.author = author;
+    public Review( Author author, int rating, String comment) {
+        this.author = new Author("","");
```

Comment:
The constructor takes in an author object, but then sets a new object of author. So author form constructor signature is unused.